### PR TITLE
Reduce duplication when using `yaml-cpp`

### PR DESCRIPTION
**Pull request by nahueespinosa**
### Proposed changes

Minor changes to the tutorial code to reduce duplication.

#### Type of change

- [ ] 🐛 Bugfix (change which fixes an issue)
- [x] 🚀 Feature (change which adds functionality)
- [ ] 📚 Documentation (change which fixes or extends documentation)

### Checklist

- [x] Lint and unit tests (if any) pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] All commits have been signed for [DCO](https://developercertificate.org/)

### CODE REVIEW

**Review by hidmic — APPROVED** [review]
Looks awesome to me. Nothing to add. I haven't tried it though. @Alondruck thoughts?

**Comment by nahueespinosa** [comment]
@Alondruck Have you had a chance to test this with the post-processing code?

**Comment by Alondruck** [comment]
> @Alondruck Have you had a chance to test this with the post-processing code?

Yes @nahueespinosa, because that I left you the comment to not initialize the vector with default values, I test the post-processing with the solution that I suggested to you and in that way it works.

**Comment by nahueespinosa** [comment]
@Alondruck You may need to submit your review. I think your comments might be in draft yet since I can't see them.

**Review by Alondruck — CHANGES_REQUESTED** [review]

File: beluga_tutorial/src/main.cpp [hunk 1]
```diff
@@ -95,111 +100,100 @@ using Particle = std::tuple<double, beluga::Weight>;
...
+  const auto parameters = YAML::LoadFile(argv[1]).as<beluga::tutorial::Parameters>();
 
   std::normal_distribution<double> initial_distribution(parameters.initial_position, parameters.initial_position_sd);
   auto particles = beluga::views::sample(initial_distribution) |                  //
                    ranges::views::transform(beluga::make_from_state<Particle>) |  //
                    ranges::views::take_exactly(parameters.number_of_particles) |  //
-                   ranges::to<beluga::TupleVector>;
+                   ranges::to<std::vector>;
+
+  std::vector<RobotRecord> records{parameters.number_of_cycles};
```
[the remark this answers is not in the document]
Comment (nahueespinosa) [reply beluga_tutorial/src/main.cpp]: Good catch! I mean to write https://github.com/Ekumen-OS/beluga/pull/383/commits/7f3523be5ea3f675f62f0d6c161805d46fd1ba3c :sweat_smile:

File: beluga_tutorial/package.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>beluga_tutorial</name>
-  <version>1.0.0</version>
+  <version>2.0.0</version>
```
Comment (nahueespinosa) [inline beluga_tutorial/package.xml:5]: All packages in the repo need to have the same version.

**Review by Alondruck — APPROVED** [review]
LGTM!

**Pull request merged**